README.md

```diff
@@ -49,7 +49,7 @@ node 'node.example.com' {
 }
 ```
 
-While the entirety of the default indicators should be reported on for maximum coverage, it may be necessarily to make exceptions for your particular environment.
+While the entirety of the default indicators should be reported on for maximum coverage, it may be necessary to make exceptions for your particular environment.
 to do this classify the array parameter indicator_exclusions with an list of all indicators you do not want to report on.
```

Comment:
You can do this by classifying the array parameter indicator_exclusions with a complete list of indicators that you don't want to report on.